.pre-commit-config.yaml

```diff
@@ -6,3 +6,13 @@ repos:
     hooks:
     - id: flake8
       additional_dependencies: [flake8-docstrings, flake8-debugger, flake8-bugbear]
+      args: [--max-complexity, "10"]
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        language_version: python3
```

Comment:
If you're adding isort you'll want to include a config for it. Satpy has a pyproject.toml where it configures it:

https://github.com/pytroll/satpy/blob/15be5c1b0c00370d7e2a827c85da25250cad0c1f/pyproject.toml#L8-L14